webpack.prod.config.js

```diff
@@ -1,5 +1,6 @@
 const path = require('path');
 const UglifyJSPlugin = require('uglifyjs-webpack-plugin'); // eslint-disable-line import/no-extraneous-dependencies
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
Can you remove the comment on line 2 and add this line at the top of the file `/* eslint import/no-extraneous-dependencies: 0 */`. Not sure if you see the linting error when you're in your editor. If you don't, perhaps check your linter settings because this rule is being violated now since we use uglify and extract text as dev dependencies.

Reply:
I did see the linting error when I checked and updated the file to fix it. Thanks for catching it!